data/human/ships.txt

```diff
@@ -2725,8 +2726,8 @@ ship "Rainmaker"
 		"heat dissipation" .6
 		"fuel capacity" 500
 		"cargo space" 25
-		"outfit space" 230
-		"weapon capacity" 60
+		"outfit space" 236
+		"weapon capacity" 66
```

Comment:
Since human ships typically have spaces in multiples of 5, I'd round up to the nearest multiple. (The Marauder Fury is fine though since it's a custom build.)
```suggestion
		"outfit space" 240
		"weapon capacity" 70
```